vipps-recurring-api-faq.md

```diff
@@ -39,6 +41,15 @@ Document version: 1.5.1.
 * [Invoicing](#invoicing)
 * [Questions?](#questions)
 
+## Does Vipps automatically create charges for an agreement?
```

Comment:
Mention the retryDays?

Add link: https://github.com/vippsas/vipps-recurring-api/blob/master/vipps-recurring-api.md#call-by-call-guide